taskcluster/ci/swift/kind.yml

```diff
@@ -51,7 +51,7 @@ tasks:
       commands:
         - ["taskcluster/scripts/build-and-test-swift.sh"]
         - [ "cd", "build/" ]
-        - [ "tar", "acf", "swift-components.tar.xz", "swift-components" ]
+        - [ "tar", "acf", "swift-components.tar.xz", "--exclude", "._*", "swift-components" ]
```

Comment:
Just to confirm, this excludes files that **start with** `._`? i.e the `.` here isn't a wild card?

Reply:
My tar manpage says "Exclude files matching PATTERN, a glob(3)-style wildcard pattern.".   I hope that that's true for OSX as well.

Reply:
Apparently it's not, because the tarball created included the apple double files.

My next try is going to move the `tar` command inside the python script.  That way we can have full control over the files that we pass to it.